src/driftpy/drift_client.py

```diff
@@ -461,6 +499,68 @@ def get_remaining_accounts_for_users(
+        remaining_accounts.append(AccountMeta(fulfillment_config.serum_bids, is_writable=True, is_signer=False))
+        remaining_accounts.append(AccountMeta(fulfillment_config.serum_asks, is_writable=True, is_signer=False))
+        remaining_accounts.append(AccountMeta(fulfillment_config.serum_base_vault, is_writable=True, is_signer=False))
+        remaining_accounts.append(AccountMeta(fulfillment_config.serum_quote_vault, is_writable=True, is_signer=False))
+        remaining_accounts.append(AccountMeta(fulfillment_config.serum_open_orders, is_writable=True, is_signer=False))
+        serum_signer_key = get_serum_signer_public_key(
+            fulfillment_config.serum_program_id,
+            fulfillment_config.serum_market,
+            fulfillment_config.serum_signer_nonce)
+        remaining_accounts.append(AccountMeta(serum_signer_key, is_writable=False, is_signer=False))
+        remaining_accounts.append(AccountMeta(self.get_signer_public_key(), is_writable=False, is_signer=False))
+        remaining_accounts.append(AccountMeta(TOKEN_PROGRAM_ID, is_writable=False, is_signer=False))
+        remaining_accounts.append(AccountMeta(self.get_spot_market_account(market_index).vault, is_writable=True, is_signer=False))
+        remaining_accounts.append(AccountMeta(self.get_spot_market_account(QUOTE_SPOT_MARKET_INDEX).vault, is_writable=True, is_signer=False))
+        remaining_accounts.append(AccountMeta(self.get_state_account().srm_vault, is_writable=False, is_signer=False))
+
+    def add_phoenix_remaining_accounts(
+        self,
+        market_index: int,
+        remaining_accounts: list[AccountMeta],
+        fulfillment_config: PhoenixV1FulfillmentConfigAccount,
+    ) -> None:
+        remaining_accounts.append(AccountMeta(fulfillment_config.pubkey, is_writable=False, is_signer=False))
+        remaining_accounts.append(AccountMeta(fulfillment_config.phoenix_program_id, is_writable=False, is_signer=False))
+        remaining_accounts.append(AccountMeta(fulfillment_config.phoenix_log_authority, is_writable=False, is_signer=False))
+        remaining_accounts.append(AccountMeta(fulfillment_config.phoenix_market, is_writable=True, is_signer=False))
+        remaining_accounts.append(AccountMeta(self.get_signer_public_key(), is_writable=False, is_signer=False))
+        remaining_accounts.append(AccountMeta(fulfillment_config.phoenix_base_vault, is_writable=True, is_signer=False))
+        remaining_accounts.append(AccountMeta(fulfillment_config.phoenix_quote_vault, is_writable=True, is_signer=False))
+        remaining_accounts.append(AccountMeta(self.get_spot_market_account(market_index).vault, is_writable=True, is_signer=False))
+        remaining_accounts.append(AccountMeta(self.get_spot_market_account(QUOTE_SPOT_MARKET_INDEX).vault, is_writable=True, is_signer=False))
+        remaining_accounts.append(AccountMeta(TOKEN_PROGRAM_ID, is_writable=False, is_signer=False))
```

Comment:
TODO: cross reference and double check with https://github.com/drift-labs/protocol-v2/blob/4eb477f24f9f03dec39e299026772f5d3c10365b/sdk/src/driftClient.ts#L3607-L3790